affiliation/token.go

```diff
@@ -0,0 +1,155 @@
+package affiliation
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"log"
+	"os"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/dgrijalva/jwt-go"
+	"github.com/google/uuid"
+)
+
+// GenerateToken ...
+func (a *Affiliation) GenerateToken() string {
```

Comment:
Is  this same as getAccessToken() at https://github.com/LF-Engineering/dev-analytics-metrics/blob/master/fixtures/service.go#L402 ?

If so, may we can move it into another common service?

Reply:
done